### PR TITLE
chore: add and update meta data

### LINKED ISSUE #18: Add app.properties
**Action required**
Add app.properties file to project.

### PR DESCRIPTION
**Pull request by onabright**
## Description
I added an app.properties file which is required for every project.

Fixes #18 
## How Has This Been Tested?
Used the instructions [provided here](https://github.com/BuildForSDG/.github/blob/master/app.properties)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by MutegekiHenry — DISMISSED** [review]

File: app.properties [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+# name of the team
+team.name: Team-229
+
+# github username
+# of the team's TTL(s)
+team.ttls: onabright
+
+# github username
+# of the team's mentor(s)
+team.mentors:
```
Comment (geneowak) [inline app.properties]: I think we are missing the team mentor details...

**Review by geneowak — CHANGES_REQUESTED** [review]

File: app.properties (hunk 1 above)
Comment (onabright) [reply app.properties]: @geneowak yes details are missing. I have just got them! Let me update

File: app.properties [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+# name of the team
+team.name: Team-229
+
+# github username
+# of the team's TTL(s)
+team.ttls: onabright
+
+# github username
+# of the team's mentor(s)
+team.mentors: Lewikeezy
```
Comment (onabright) [inline app.properties:88]: I have added the team mentor's Github handle

**Review by onabright — COMMENTED** [review]
I have added the team mentor's details.

**Review by MutegekiHenry — APPROVED** [review]

**Review by geneowak — APPROVED** [review]

**Pull request merged**